pkg/certs/rotate.go

```diff
@@ -233,7 +237,11 @@ func patchSecret(ctx context.Context, secretNamespace, secretName, pkiPath strin
 	for k, v := range certMap {
 		d, err := os.ReadFile(filepath.Join(pkiPath, k))
 		if err != nil {
-			return fmt.Errorf("reading file %s: %w", k, err)
+			if !errors.Is(err, os.ErrNotExist) {
+				return fmt.Errorf("reading file %s: %w", k, err)
```

Comment:
(suggestion, minor) : WDTY mentioning the full path of the file that in the error : 
```suggestion
				return fmt.Errorf("reading file %s: %w",  filepath.Join(pkiPath, k), err)
```
We can use a variable to define the full path of the file if we choose that option.